opa-audit-operator/tests/integration/test_charm.py

```diff
@@ -0,0 +1,119 @@
+        api_instance = client.RbacAuthorizationV1Api(api_client)
+        try:
+            for k8s_obj in role_binding:
+                if k8s_obj["kind"] == "ClusterRoleBinding":
+                    api_instance.create_cluster_role_binding(body=k8s_obj)
+                if k8s_obj["kind"] == "ClusterRole":
+                    api_instance.create_cluster_role(body=k8s_obj)
+        except ApiException as err:
+            if err.status == 409:
+                # ignore "already exists" errors so that we can recover from
+                # partially failed setups
+                return
+            else:
+                raise
+    for series in meta["series"]:
+        await ops_test.model.deploy(charm, application_name=series, series=series)
+    await ops_test.model.wait_for_idle(wait_for_active=True, timeout=60 * 60)
+
+    with open("docs/gatekeeper-rb.yaml.template", "r") as fh:
+        template = Template(fh.read())
+        role_binding_file.write_text(
+            template.render(
+                service_account_user=f"system:serviceaccount:{model_name}:kubernetes-operator"
+            )
+        )
+    role_binding = yaml.load_all(role_binding_file.read_text(), Loader=yaml.FullLoader)
+    # yield role_binding
+    # import pdb; pdb.set_trace()
+    with client.ApiClient() as api_client:
+        api_instance = client.RbacAuthorizationV1Api(api_client)
+        try:
+            for k8s_obj in role_binding:
+                if k8s_obj["kind"] == "ClusterRoleBinding":
+                    api_instance.create_cluster_role_binding(body=k8s_obj)
+                if k8s_obj["kind"] == "ClusterRole":
+                    api_instance.create_cluster_role(body=k8s_obj)
+        except ApiException as err:
+            if err.status == 409:
+                # ignore "already exists" errors so that we can recover from
+                # partially failed setups
+                return
+            else:
+                raise
+    ca_cert = ""
+    with client.ApiClient() as api_client:
+        # Create an instance of the API class
+        api_instance = client.CoreV1Api(api_client)
+        name = f"{model_name}-gatekeeper-validating-webhook-configuration"
+
+        ca_cert = api_instance.read_namespaced_secret(name, model_name)
+
+    with client.ApiClient() as api_client:
+        # Create an instance of the API class
+        api_instance = client.AdmissionregistrationV1Api(api_client)
+        name = f"{model_name}-gatekeeper-validating-webhook-configuration"
+        for i in range(1):
+            body = [
+                {
+                    "op": "replace",
+                    "path": f"/webhooks/{i}/clientConfig/caBundle",
+                    "value": f"{ca_cert}",
+                }
+            ]
+
+            api_instance.patch_validating_webhook_configuration(name, body)
+
+
+async def test_status_messages(ops_test):
+    """ Validate that the status messages are correct. """
+    expected_messages = {}
```

Comment:
Given that expected_messages is empty, this test appears to be a no-op. Either add appropriate statuses in here, or delete the test.

Reply:
Please update or remove this test.

Reply:
:+1: